### PR TITLE
[IMPAC-616] Stacking columns for cash projection overdue invoices

### PR DESCRIPTION
**Pull request by xaun**
screen shot 2017-09-01 at 17 51 45

### CODE REVIEW

**Comment by xaun** [comment]
https://github.com/maestrano/impac-finance-bolt/pull/43

File: src/components/widgets/accounts-cash-projection/accounts-cash-projection.directive.coffee [hunk 1]
```diff
@@ -71,10 +74,12 @@ module.controller('WidgetAccountsCashProjectionCtrl', ($scope, $q, $filter, Impa
         legend:
           useHTML: true
           labelFormatter: ->
-            name = this.name
-            imgSrc = ImpacAssets.get(_.camelCase(name + 'LegendIcon'))
+            filename = this.name
```
Comment (cesar-tonnoir) [inline src/components/widgets/accounts-cash-projection/accounts-cash-projection.directive.coffee]: Please show the legend only for the main series: "Payables" and "Receivables - no need to have x6 legend entries as the tooltip should differentiate the different stacks.

**Comment by xaun** [comment]
@cesar-tonnoir this is ready for review

screen shot 2017-09-20 at 20 52 46

**Comment by cesar-tonnoir** [comment]
@xaun I've changed the base back to release/v1.6.1: "staging" you don't need reviews to merge to "staging" as this is a branch we only use to deploy and test in UAT. "release/a.b.c" branches however, aim to merge code to regular versioned branches

**Pull request closed without merging**